### PR TITLE
Changes and fixes up until May 17th 2019

### PR DESCRIPTION
**Pull request by syversenkr**
Consolidation of compatible changes and fixes up until May 17th 2019, used in Norway from August 2019.
For manual rebasing of CEN NeTEx master per NeTEx spec v1.1 document updates of late (August/November) 2019.

### CODE REVIEW

File: xsd/NeTEx_publication.xsd [hunk 1]
```diff
@@ -1595,26 +1700,26 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
...
 			</xsd:annotation>
 			<xsd:selector xpath=".//netex:Equipment"/>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
-		</xsd:unique>
-		<!-- =====Equipment Key ========================== -->
-		<xsd:keyref name="Equipment_KeyRef" refer="netex:Equipment_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:EquipmentRef"/>
+		</xsd:unique>-->
+	   <!-- === (GENERAL) Equipment Key ========================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:1712]: Indent error

File: xsd/NeTEx_publication.xsd [hunk 2]
```diff
@@ -3066,17 +3171,17 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====Entrance============================== -->
+		<!-- =====Entrance==============================
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:3174]: Why is this commented out?

File: xsd/NeTEx_publication.xsd [hunk 3]
```diff
@@ -3435,6 +3610,30 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
+		
+<!-- =====ParkingProperties============================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:3614]: Indent error

File: xsd/NeTEx_publication.xsd [hunk 4]
```diff
@@ -3837,7 +4036,7 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 		</xsd:unique>
 		<!-- =====RailwayJunction Key ========================== -->
 		<xsd:keyref name="RailwayJunction_KeyRef" refer="netex:RailwayJunction_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:RailwayJunctionRef | .//netex:RailwayPointRef | .//netex:RailwayElement/netex:FromPointRef | .//netex:RailwayElement/netex:ToPointRef"/>
+			<xsd:selector xpath=" .//netex:RailwayPointRef | .//netex:RailwayElement/netex:FromPointRef | .//netex:RailwayElement/netex:ToPointRef"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:4039]: Space after xpath="

File: xsd/NeTEx_publication.xsd [hunk 5]
```diff
@@ -5268,49 +5467,10 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@version"/>
 		</xsd:keyref>
 		<xsd:key name="ServicePattern_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:ServicePattern |.//netex:DeadRunPattern "/>
+			<xsd:selector xpath=".//netex:ServicePattern |.//netex:DeadRunPattern"/>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====PointInJourneyPatternLink============================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:5275]: Just wondering where did go to...

File: xsd/NeTEx_publication.xsd [hunk 6]
```diff
@@ -6365,30 +6525,18 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@version"/>
 		</xsd:keyref>
 		<xsd:key name="BorderPoint_AnyVersionedKey">
-			<xsd:selector xpath=".//netex:BorderPoint  "/>
+			<xsd:selector xpath=".//netex:BorderPoint"/>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
 		<!-- =====FareProduct============================== -->
-		<!-- =====FareProduct Key ========================== -->
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:6373]: Where did this go to?

File: xsd/NeTEx_publication.xsd [hunk 7]
```diff
@@ -7241,7 +7431,7 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:annotation>
 				<xsd:documentation>Every [FareTable Id + Version] must be unique within document.</xsd:documentation>
 			</xsd:annotation>
-			<xsd:selector xpath=".//netex:FareTable"/>
+			<xsd:selector xpath=".//netex:FareTable   | .//netex:StandardFareTable"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:7434]: Many of the Fare selectors are using multiple spaces around |

File: xsd/NeTEx_publication.xsd [hunk 8]
```diff
@@ -7495,6 +7707,26 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
...
+		<xsd:unique name="EligibilityChangePolicy">
+			<xsd:annotation>
+				<xsd:documentation>Every [EligibilityChangePolicy Id + Version] must be unique within document.</xsd:documentation>
+			</xsd:annotation>
+			<xsd:selector xpath=".//netex:EligibilityChangePolicy"/>
+			<xsd:field xpath="@id"/>
+			<xsd:field xpath="@version"/>
+		</xsd:unique>
+		<xsd:keyref name="EligibilityChangePolicy_KeyRef" refer="netex:EligibilityChangePolicy_AnyVersionedKey">
+			<xsd:selector xpath=" .//netex:EligibilityChangePolicyRef"/>
```
Comment (skinkie) [inline xsd/NeTEx_publication.xsd:7721]: space after xpath="

File: xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd [hunk 9]
```diff
@@ -118,18 +132,6 @@ Rail transport, Roads and Road transport
 		</xsd:sequence>
 	</xsd:group>
 	<!-- =====GROUPING=============================================== -->
-	<xsd:complexType name="groupOfEntitiesInFrame_RelStructure">
```
Comment (skinkie) [inline xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd:121]: So where did this go?

File: xsd/NeTEx_publication.xsd (hunk 2 above)
Comment (Aurige) [reply xsd/NeTEx_publication.xsd:3174]: looks like a bug ...

**Comment by Aurige** [comment]
I did a quick walk throug but didn't spot other points than Stefan ! So most important seems to ba a cleaning of some trailing spaces in NeTEx_publication.xsd

File: xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd (hunk 9 above)
Comment (Aurige) [reply xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd:121]: line 67

**Comment by syversenkr** [comment]
I can fix the spottet errors in NeTEx_publication.xsd and add the update to the pull in a little while

File: xsd/NeTEx_publication.xsd (hunk 6 above)
Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:6373]: This should be looked into, as it seems the refactoring at least have missed this concept in the NeTEx_publication_timetable.xsd
See output from 
  grep -irHn5 "FareProduct Key" *
for v1.09 and v1.1 (May 17th 2019).

Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:6373]: (file attachement failed)

**v1.0:**

xsd/NeTEx_publication.xsd-6368-			<xsd:selector xpath=".//netex:BorderPoint  "/>
xsd/NeTEx_publication.xsd-6369-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication.xsd-6370-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication.xsd-6371-		</xsd:key>
xsd/NeTEx_publication.xsd-6372-		
xsd/NeTEx_publication.xsd:6373:		
xsd/NeTEx_publication.xsd-6374-		
xsd/NeTEx_publication.xsd-6480-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
xsd/NeTEx_publication.xsd-6481-			<xsd:selector xpath=".//netex:PreassignedFareProductRef"/>
xsd/NeTEx_publication.xsd-6482-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication.xsd-6483-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication.xsd-6484-		</xsd:keyref>
--
--
xsd/NeTEx_publication_timetable.xsd-6382-			<xsd:selector xpath=".//netex:BorderPoint  "/>
xsd/NeTEx_publication_timetable.xsd-6383-			<xsd:field xpath="@id"/>
xsd/NeTEx_publication_timetable.xsd-6384-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication_timetable.xsd-6385-		</xsd:key>
xsd/NeTEx_publication_timetable.xsd-6386-		
xsd/NeTEx_publication_timetable.xsd:6387:		
xsd/NeTEx_publication_timetable.xsd-6388-		
xsd/NeTEx_publication_timetable.xsd-6494-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
xsd/NeTEx_publication_timetable.xsd-6495-			<xsd:selector xpath=".//netex:PreassignedFareProductRef"/>
xsd/NeTEx_publication_timetable.xsd-6496-			<xsd:field xpath="@ref"/>
xsd/NeTEx_publication_timetable.xsd-6497-			<xsd:field xpath="@version"/>
xsd/NeTEx_publication_timetable.xsd-6498-		</xsd:keyref>

Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:6373]: v1.1 (per 17th may 2019)

```
NeTEx_publication.xsd-6622-			</xsd:annotation>
NeTEx_publication.xsd-6623-			<xsd:selector xpath=".//netex:PreassignedFareProduct"/>
NeTEx_publication.xsd-6624-			<xsd:field xpath="@id"/>
NeTEx_publication.xsd-6625-			<xsd:field xpath="@version"/>
NeTEx_publication.xsd-6626-		</xsd:unique>
NeTEx_publication.xsd:6627:		<!-- =====PreassignedFareProduct Key ========================== -->
NeTEx_publication.xsd-6628-		<xsd:keyref name="PreassignedFareProduct_KeyRef" refer="netex:PreassignedFareProduct_AnyVersionedKey">
NeTEx_publication.xsd-6629-			<xsd:selector xpath=".//netex:PreassignedFareProductRef | .//netex:SupplementToFareProductRef"/>
NeTEx_publication.xsd-6630-			<xsd:field xpath="@ref"/>
NeTEx_publication.xsd-6631-			<xsd:field xpath="@version"/>
NeTEx_publication.xsd-6632-		</xsd:keyref>
```

File: xsd/NeTEx_publication.xsd (hunk 5 above)
Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:5275]: Moved to NeTEx_publication_timetable.xsd:5316
(diff v1.0 vs v1.1 are similar, but the concept is removed from xsd/NeTEx_publication_timetable.xsd in v1.1)

File: xsd/NeTEx_publication.xsd (hunk 2 above)
Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:3174]: Will assume an error, as 
`xsd:element name="Entrance" type="SiteEntrance_VersionStructure" abstract="false" substitutionGroup="SiteComponent">`
is defined in netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd:337

REMARK: Have kept the comment regardless, someone (@nick-knowles ?) should review and (maybe) revert to again include the 
`<xsd:keyref name="Entrance_AnyKeyRef" refer="netex:Entrance_AnyVersionedKey">`

File: xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd (hunk 9 above)
Comment (syversenkr) [reply xsd/netex_framework/netex_genericFramework/netex_grouping_version.xsd:121]: Moved to netex_framework/netex_genericFramework/netex_grouping_version.xsd:67
(except that, contents of v1.0 and v.1.1 seems to be the same)

File: xsd/NeTEx_publication.xsd (hunk 7 above)
Comment (syversenkr) [reply xsd/NeTEx_publication.xsd:7440]: Have also corrected these where spottet, but probably heaps more in the code. Though cosmetic, does not affect the functionality, these and similar issues should be cleaned up iterative when spotted.

**Comment by Aurige** [comment]
Ok, so shall we merge this version ?

**Comment by syversenkr** [comment]
Yes indeed, I believe we should.

**Pull request merged**